packages/grafana-data/src/utils/processDataFrame.test.ts

```diff
@@ -135,6 +146,16 @@ describe('SerisData backwards compatibility', () => {
     expect(roundtrip).toMatchObject(table);
   });
 
+  it('Can convert empty TableData to DataFrame', () => {
```

Comment:
```suggestion
  it('converts empty TableData to DataFrame', () => {
```

Reply:
prefer the BDD naming pattern to make proper sentences describe("When xxx") , it("Should", or it("Can

Reply:
maybe should be lowercase C though

Reply:
Just trying to follow the convention used in other tests in this suite :) But i'm totally fine with `can...` if we update all of them